app/exec_cmd.go

```diff
@@ -33,8 +36,36 @@ func (e *execCmd) Run(l *ui.UI, cache *cache.Cache, sta *state.State, globalStat
 		return errors.WithStack(err)
 	}
 
+	githubToken := os.Getenv("HERMIT_GITHUB_TOKEN")
+	if githubToken == "" {
+		githubToken = os.Getenv("GITHUB_TOKEN")
+		l.Tracef("GitHub token set from GITHUB_TOKEN")
+	} else {
+		l.Tracef("GitHub token set from HERMIT_GITHUB_TOKEN")
+	}
+
+	var (
+		ghClient        = github.New(defaultHTTPClient, githubToken)
```

Comment:
This should be injected from main into Run, and the getenv code removed from here.